go.mod

```diff
@@ -54,13 +54,13 @@ require (
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
 
 	//pinned k8s.io to v0.26.1 tag
-	k8s.io/api v0.26.1
+	k8s.io/api v0.26.3
 	k8s.io/apiextensions-apiserver v0.26.1
-	k8s.io/apimachinery v0.26.1
+	k8s.io/apimachinery v0.26.3
```

Comment:
If we are changing one k8s deps I think we should change others as well. For example `apiextensions-apiserver`, `client-go`, cli-runtime` etc. 
What do you think?

Reply:
These were automatically pulled when the openshift/client-go package was updated.
It actually makes sense to put all of the k8s deps in a separate PR altogether. Let me create a separate PR for the k8s deps.